cmd/wallet-sdk-gomobile/api/metrics.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
+
+import goapi "github.com/trustbloc/wallet-sdk/pkg/api"
+
+// A MetricsEvent represents a single event that occurred that had performance metrics taken.
+type MetricsEvent struct {
+	// GoAPIMetricsEvent will not be accessible directly in the bindings (will be "skipped").
+	// This is OK - this field is not intended to be set directly by the user of the SDK.
+	GoAPIMetricsEvent *goapi.MetricsEvent
+}
+
+// Event returns the event name.
+func (m *MetricsEvent) Event() string {
```

Comment:
Just curious if you think it'd be cleaner to have function names start with a verb to better represent what they're doing? `getEvent()`, etc

Reply:
My thinking was that for a function that's a simple "getter" type of operation that doesn't require DB lookups or other more complex operations, that a simple noun should suffice. If this call required a database lookup or something like that, I think `GetEvent()` would make more sense. Regardless of that, currently in the SDK all the other methods that do similar things (simple field returns) don't use `get`, so I think it's best to keep it consistent. Thanks for providing the suggestion, though - let me know what you think of my reasonin

Reply:
Fair enough, thanks @DRK3! 👍